src/systems/filecoin_mining/storage_mining/storage_miner_actor.id

```diff
@@ -21,51 +21,76 @@ type SectorExpirationQueue struct {
     Remove(n sector.SectorNumber)
 }
 
+type SectorTable struct {
+    SectorSize         sector.SectorSize
+    ActiveSectors      util.UVarint
+    CommittedSectors   util.UVarint
+    RecoveringSectors  util.UVarint
+    FailingSectors     util.UVarint
+    TerminatedSectors  util.UVarint  // transient State that get reset on every constructPowerReport
```

Comment:
this is a bit annoying because it is too similar to an additional state, and that creates more complexity. maybe change it to:

```go
TerminationFaults util.UVarint
// or
TerminationFaultCount util.UVarint
```